server/routes/github.js

```diff
@@ -0,0 +1,128 @@
+const express = require("express");
+const fetch = require('node-fetch');
+const mongoose = require("mongoose");
+const Post = require("../models/Post");
+
+const mongoConnectionURL = process.env.MONGODB_SRV; 
+
+const connectToDB = async () => {
+    mongoose
+    .connect(mongoConnectionURL, {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        useCreateIndex: true,
+        dbName: "Dashboard",
+    })
+    .then(() => console.log("Connected to MongoDB"))
+    .catch(err => console.log(`${err}: Failed to connect to MongoDB`));
+}
+
+
+const issues = "issues";
+const pullRequests = "pulls";
+
+/*
+org and repo are hard-coded here, will need input from frontend to pass in params
+*/
+var org = "MLH-Fellowship";
+var repo = "httpie";
+
+async function fetchIssues(org, repo) {
```

Comment:
Following the comment above, if we were to expand this to be a tool for an API endpoint, then maybe a creator param that defaults to server would be good.

Reply:
Yepp agree - issues/PR should have creator as the user once they manually create a post using the add form. Contacts has creator as server since it will be create pre-populated